tests/network/sriov.go

```diff
@@ -250,17 +252,14 @@ var _ = Describe("SRIOV", Serial, decorators.SRIOV, func() {
 			Expect(checkDefaultInterfaceInPod(vmi)).To(Succeed())
 
 			By("checking virtual machine instance has two interfaces")
-			checkInterfacesInGuest(vmi, []string{"eth0", "eth1"})
+			expectedInterfaces := []string{"eth0", "eth1"}
+			checkInterfacesInGuest(vmi, expectedInterfaces)
 
-			domSpec, err := tests.GetRunningVMIDomainSpec(vmi)
-			Expect(err).ToNot(HaveOccurred())
-			rootPortController := []api.Controller{}
-			for _, c := range domSpec.Devices.Controllers {
-				if c.Model == "pcie-root-port" {
-					rootPortController = append(rootPortController, c)
-				}
+			for _, iface := range expectedInterfaces {
+				onRootPCIBridge, err := isInterfaceOnRootPCIComplex(vmi, iface)
```

Comment:
nit: There are two terms here: `RootPCIBridge` and `RootPCIComplex`, please select one for consistency.
Same bellow.

Reply:
Thanks.
The following assertion still uses the previous terminology in the error message.
```
Expect(onRootPCIComplex).To(BeTrue(), fmt.Sprintf("Expected interface %s on PCI root bridge", iface))
```